encoding.bs

```diff
@@ -995,7 +971,34 @@ steps:
 <p>To <dfn export>UTF-8 encode</dfn> a scalar value stream <var>stream</var>, return the result of
 <a lt=encode for=/>encoding</a> <var>stream</var> using encoding <a>UTF-8</a>.
 
+<hr>
+
+<p>To <dfn export>BOM sniff</dfn> a byte stream <var>stream</var>, run these steps:
+
+<ol>
+  <li><p>Wait until <var>stream</var> has three bytes available or the <a>end-of-stream</a> has been
```

Comment:
One space for markup indentation here and below. (And more indentation for the children of the table, to match what it was before.